src/irmin/type/type.mli

```diff
@@ -298,14 +298,17 @@ type 'a pp = 'a Fmt.t
 type 'a of_string = string -> ('a, [ `Msg of string ]) result
 (** The type for parsers. *)
 
-val pp : 'a t -> 'a pp
+val pp : ?ocaml_syntax:bool -> 'a t -> 'a pp
 (** [pp t] is the pretty-printer for values of type [t]. *)
```

Comment:
Documentation for `~ocaml_syntax` needed here, including at least a specification of the default. (I'd be in favour of making it `?ocaml_syntax:unit` to remove that ambiguity altogether.)